.github/workflows/example-get-started-test.yaml

```diff
@@ -0,0 +1,26 @@
+name: example-get-started test
+on: 
+  push:
+    paths:
+      - example-get-started/**
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+        cache: 'pip'
+    - uses: aws-actions/configure-aws-credentials@v2
+      with:
+        aws-region: us-east-2
+        role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
```

Comment:
Added to repo secrets

Reply:
Is it the sandbox I assume?

Reply:
Wouldn't you need to add this repo in https://github.com/iterative/itops/blob/dd03f7752ee61f09487902ce1e4e1d1074393748/terraform/iterative/iam_sandbox.tf#L38 ?

Reply:
Yup, thanks. I was sure I missed something (probably more). That's why I'm starting small.